### PR TITLE
Fix the `README.md` file

### PR DESCRIPTION
**Pull request by daoanhhuy26012001**
- [x] What is the purpose of the `Introduce` section?
- [x] Unclear targets.
- [x] For the Technical section, we should have a short introduction for each technical item we use. I think it would be better to add an official site for reference.
- [x] Guideline does not work.

### CODE REVIEW

File: javascript/practice-two/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ## Overview
 
 - Introduce:
-  - Build grocery bud with functions like add, update, delete, clear list.
+  - Build a grocery bud app to list the store's products and functions for users to add, update, delete, clear.
```
Comment (phuocduong-agilityio) [inline javascript/practice-two/README.md:8]: It's not clear as old version. You should clear what this project for, introduce some infor about page, what is the purposes, not features. 
Ex: A landing page for introducing AgilityIO company

File: javascript/practice-two/README.md [hunk 2]
```diff
@@ -17,14 +17,22 @@
 ## Target
 
 - DOM manipulation.
+  - Using methods: `createAttribute()`, `setAttributeNode()` and `appendChild()`
```
Comment (phuocduong-agilityio) [inline javascript/practice-two/README.md:20]: This is not really a document for project/practice. Please focus on target after finishing practice. Not details about tech that you used

File: javascript/practice-two/README.md [hunk 3]
```diff
@@ -17,14 +17,22 @@
...
-- Parcel.
-- JavaScript.
+- [HTML5](https://www.w3schools.com/html/default.asp)/ [CSS3](https://www.w3schools.com/css/default.asp):
+  - In this exercise, use HTML5 to create the blocks or elements needed to build a website
+  - Then use CSS3 formatting language to design for the best user experience.
+- [Parcel](https://parceljs.org/):
+  - Parcel is a build tool and bundler for web applications. It helps automate the web application building process by automatically resolving and packaging project files, resources, and dependencies into static files that are optimized for deployment.
+- [JavaScript](https://www.w3schools.com/js/):
+  - Use this language to handle the logic for the project.(e.g.: the add an item function, handle event,...)
+  -
```
Comment (phuocduong-agilityio) [inline javascript/practice-two/README.md:35]: Remove this

File: javascript/practice-two/README.md (hunk 1 above)
Comment (daoanhhuy26012001) [reply javascript/practice-two/README.md]: Done!

File: javascript/practice-two/README.md (hunk 2 above)
Comment (daoanhhuy26012001) [reply javascript/practice-two/README.md]: Done!

File: javascript/practice-two/README.md (hunk 3 above)
Comment (daoanhhuy26012001) [reply javascript/practice-two/README.md:36]: Done

File: javascript/practice-two/README.md [hunk 4]
```diff
@@ -17,14 +17,22 @@
...
+- [Good structure project](https://www.rahmandawibowo.com/blog/en/how-to-manage-your-folder-structure-in-js-project-effectively/).
+  - Organize your files in a clear and structured manner, separating HTML, CSS, and JavaScript code into different files.
+  - Consider using a modular approach by dividing your JavaScript code into separate modules or components for better maintainability and reusability.
 
 ## Technical
 
-- HTML5/ CSS3.
-- Parcel.
-- JavaScript.
+- [HTML5](https://www.w3schools.com/html/default.asp)/ [CSS3](https://www.w3schools.com/css/default.asp):
```
Comment (phattruong-agilityio) [inline javascript/practice-two/README.md:29]: New line for CSS3

File: javascript/practice-two/README.md [hunk 5]
```diff
@@ -17,14 +17,22 @@
...
+  - In this exercise, use HTML5 to create the blocks or elements needed to build a website
+  - Then use CSS3 formatting language to design for the best user experience.
+- [Parcel](https://parceljs.org/):
+  - Parcel is a build tool and bundler for web applications. It helps automate the web application building process by automatically resolving and packaging project files, resources, and dependencies into static files that are optimized for deployment.
+- [JavaScript](https://www.w3schools.com/js/):
+  - Use this language to handle the logic for the project.(e.g.: the add an item function, handle event,...)
```
Comment (phattruong-agilityio) [inline javascript/practice-two/README.md:35]: Short description about this language

File: javascript/practice-two/README.md [hunk 6]
```diff
@@ -5,7 +5,7 @@
 ## Overview
 
 - Introduce:
-  - Build grocery bud with functions like add, update, delete, clear list.
+  - Build a page to manage the goods of the grocery store 'Grocery Bud'.
```
Comment (phattruong-agilityio) [inline javascript/practice-two/README.md:8]: Build a web-app to...

**Pull request merged**